### PR TITLE
[Quickfix] Use entypo-mail on mobile view

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Comment by svbergerem** [comment]
Did you actually test this?

header

I also checked other places (e.g. the profile) and I prefer the entypo icon.

**Comment by AugierLe42e** [comment]
> I prefer the entypo icon.

Well, it's weird it is not the same on mobile and on desktop. All other icons are stricktly the same.

File: app/assets/stylesheets/header.scss [hunk 1]
```diff
@@ -116,11 +116,15 @@ header > .dark-header > nav {
       }
     }
   }
-  [class^="entypo-"], [class*="entypo-"] {
+  [class^="entypo-"],
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/header.scss]: Selector should have depth of applicability no greater than 3, but was 4

File: app/assets/stylesheets/header.scss [hunk 2]
```diff
@@ -116,11 +116,15 @@ header > .dark-header > nav {
...
+  [class*="entypo-"],
+  .diaspora-custom-mail {
     color: inherit;
     font-size: $font-size-h3;
     vertical-align: middle;
   }
+  .diaspora-custom-mail { font-size: 26px; }
```
Comment (diaspora-code-review) [inline app/assets/stylesheets/header.scss]: Selector should have depth of applicability no greater than 3, but was 4

**Comment by AugierLe42e** [comment]
> Did you actually test this?

Nope, sorry. Ninja commit before going to school this morning. Fixed it.

header

**Comment by AugierLe42e** [comment]
Ready to merge.

**Comment by svbergerem** [comment]
bildschirmfoto am 2015-10-11 um 12 59 33
bildschirmfoto am 2015-10-11 um 12 56 06

bildschirmfoto am 2015-10-11 um 12 59 08

bildschirmfoto am 2015-10-11 um 12 55 21

bildschirmfoto am 2015-10-11 um 12 58 49
bildschirmfoto am 2015-10-11 um 12 55 41

> I prefer the entypo icon

**Comment by AugierLe42e** [comment]
> I prefer the entypo icon

Yeah, but there's not consistancy, then :/

**Comment by svbergerem** [comment]
You could also change the icon for the mobile view if you want it to be consistent.

**Comment by AugierLe42e** [comment]
> You could also change the icon for the mobile view if you want it to be consistent.

Which one?

**Comment by svbergerem** [comment]
> Which one?

Whatever icon you want to be consistent. In this case it would be the mobile conversations icon.

**Comment by jaywink** [comment]
I have to say I prefer the current one too, sorry. Maybe make elsewhere consistent to use the one we have now everywhere instead of changing the icon?

**Comment by AugierLe42e** [comment]
> Maybe make elsewhere consistent to use the one we have now everywhere instead of changing the icon?

Well, when I reworked the mobile, some said me they prefered the old mail icon to the entypo one. I don't care as long as it is the same icon both on mobile and desktop version, but you guys have to choose. This is the kind of details that make the design look messy.

**Comment by SuperTux88** [comment]
I like the entypo icon more than the other.

**Comment by AugierLe42e** [comment]
Well let's adopt the entypo icon for mobile skin instead, then. But let me remind that the old icon on mobile version wasn't replaced because [I was requested not to](https://github.com/diaspora/diaspora/pull/6285#issuecomment-128158402) ;)

**Comment by svbergerem** [comment]
Keeping the old icons was a good trade-off in that PR since it could have completely blocked it. This PR is only about changing the icon so let's see what @Flaburgan has to say.

**Comment by Flaburgan** [comment]
This is only an icon, I don't wanna fight about, pick the one you want ;)

**Comment by AugierLe42e** [comment]
screen shot 2015-10-24 at 18 04 25

**Comment by AugierLe42e** [comment]
Ready to merge.

**Comment by svbergerem** [comment]
Since the custom mail icon is not used anymore you could also drop it from `app/assets/stylesheets/typography.scss`, `app/assets/images/fonts/svg-icons/`and maybe even remove it from `diaspora-custom.ttf`.

**Comment by AugierLe42e** [comment]
Done. I prefer not to touch `diaspora-custom.ttf` because I don't remember on which site I've generated it.

**Comment by svbergerem** [comment]
Thank you!

**Pull request merged**